src/main/java/org/team1540/robot2024/RobotContainer.java

```diff
@@ -95,6 +99,22 @@ public RobotContainer() {
                         drivetrain::addVisionMeasurement,
                         () -> 0.0, // TODO: ACTUALLY GET ELEVATOR HEIGHT HERE
                         new VisionPoseAcceptor(drivetrain::getChassisSpeeds, () -> 0.0)); // TODO: ACTUALLY GET ELEVATOR VELOCITY HERE
+
+                PathPlannerLogging.setLogCurrentPoseCallback((pose) -> {
+                    Logger.recordOutput("PathPlanner/Position", pose);
+                });
+                PathPlannerLogging.setLogTargetPoseCallback((pose) -> {
+                    Logger.recordOutput("PathPlanner/TargetPosition", pose);
+                });
+                PathPlannerLogging.setLogActivePathCallback((path) -> {
+                    if(path.hashCode() != currentPathHash){
```

Comment:
why do you check so aggressively for if it's changed? Also does it make sense to put this in robotcontainer and not AutoManager, it doesn't look like it uses anything that robotcontainer creates?